pages/apis/graphql/graphql_cookbook.md

```diff
@@ -541,6 +541,64 @@ mutation {
 }
 ```
 
+## Clusters
+
+A collection of common tasks with clusters using the GraphQL API.
+
+### List cluster queues 
+
+List the first 20 cluster queues' key and ID for the first 10 clusters in the organisation
+
+```graphql
+query getClusterQueues {
+  organization(slug: "organization-slug") {
+    clusters(first:10){
+      edges{
+        node{
+          name
+          queues(first:20){
+            edges{
+              node{
+                key
+                id
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+### List jobs as part of a specific cluster queue
```

Comment:
Does this still keep the right meaning?

```suggestion
### List jobs in a cluster queue
```

Just looking if there's a sensible way to shorten it a bit.

Reply:
In the latest commit 👍, TY @mbelton-buildkite